diff --git a/node_modules/eris/lib/structures/AutocompleteInteraction.js b/node_modules/eris/lib/structures/AutocompleteInteraction.js
index fa9bcbc..95a2d78 100644
--- a/node_modules/eris/lib/structures/AutocompleteInteraction.js
+++ b/node_modules/eris/lib/structures/AutocompleteInteraction.js
@@ -17,7 +17,7 @@ const {InteractionResponseTypes} = require("../Constants");
 * @prop {Array<Object>?} data.options The run Application Command options
 * @prop {String} data.options[].name The name of the Application Command option
 * @prop {Number} data.options[].type Command option type, 1-10
-* @prop {String? | Number? | Boolean?} data.options[].value The value of the run Application Command (Mutually exclusive with options)
+* @prop {(String | Number | Boolean)?} data.options[].value The value of the run Application Command (Mutually exclusive with options)
 * @prop {Boolean?} data.options[].focused Whether or not the option is focused
 * @prop {Array<Object>?} data.options[].options The run Application Command options (Mutually exclusive with value)
 * @prop {String?} guildID The ID of the guild in which the interaction was created
diff --git a/node_modules/eris/lib/structures/CommandInteraction.js b/node_modules/eris/lib/structures/CommandInteraction.js
index 439bc3f..3f870c9 100644
--- a/node_modules/eris/lib/structures/CommandInteraction.js
+++ b/node_modules/eris/lib/structures/CommandInteraction.js
@@ -22,7 +22,7 @@ const {InteractionResponseTypes} = require("../Constants");
 * @prop {Array<Object>?} data.options The run Application Command options
 * @prop {String} data.options[].name The name of the Application Command option
 * @prop {Number} data.options[].type Command option type, 1-10
-* @prop {String? | Number? | Boolean?} data.options[].value The value of the run Application Command (Mutually exclusive with options)
+* @prop {(String | Number | Boolean)?} data.options[].value The value of the run Application Command (Mutually exclusive with options)
 * @prop {Array<Object>?} data.options[].options The run Application Command options (Mutually exclusive with value)
 * @prop {Object?} data.resolved converted users + roles + channels
 * @prop {Collection<User>?} data.resolved.users converted users
diff --git a/node_modules/eris/lib/structures/UnknownInteraction.js b/node_modules/eris/lib/structures/UnknownInteraction.js
index 4e27dee..4eb5940 100644
--- a/node_modules/eris/lib/structures/UnknownInteraction.js
+++ b/node_modules/eris/lib/structures/UnknownInteraction.js
@@ -9,7 +9,7 @@ const {InteractionResponseTypes} = require("../Constants");
 * Represents an unknown interaction. See Interaction for more properties.
 * Note: Methods are not guaranteed to work properly, they are all added just in case you know which to use.
 * @extends Interaction
-* @prop {PrivateChannel? | TextChannel? | NewsChannel?} channel The channel the interaction was created in. Can be partial with only the id if the channel is not cached.
+* @prop {(PrivateChannel | TextChannel | NewsChannel)?} channel The channel the interaction was created in. Can be partial with only the id if the channel is not cached.
 * @prop {Object?} data The data attached to the interaction
 * @prop {String?} guildID The ID of the guild in which the interaction was created
 * @prop {Member?} member The member who triggered the interaction (This is only sent when the interaction is invoked within a guild)
